apps/vscode-extension/src/utils/files/updateFilesDataElapsedTime.ts

```
@@ -1,18 +1,21 @@
+    const fileData = filesData[filePath];
+    // Only use frozenTime if it is not null (0 is valid)
     fileData.elapsedTime =
-      fileData.isFrozen && fileData.frozenTime
+      fileData.isFrozen && fileData.frozenTime !== null
         ? fileData.frozenTime
         : Math.floor((now - fileData.startTime) / 1000);
   });
 
   return filesData;
+
+  return filesData;
```

Biome (2.1.2)

[error] This code will never be reached ...

... because this statement will return from the function beforehand


(lint/correctness/noUnreachable)